.pre-commit-config.yaml

```diff
@@ -52,6 +52,18 @@ repos:
   rev: 6.1.0
   hooks:
   - id: flake8
+    additional_dependencies:
+    - flake8-cognitive-complexity ~= 0.1.0
+    - mccabe
```

Comment:
`mccabe` is already included with `flake8` and is declared as a dependency. You don't need to depend on it additionally.

Reply:
Removed

Reply:
Marking as resolved.